### PR TITLE
Document ES5 adapter in README.

### PR DESCRIPTION
**Pull request by aomarks**
- [x] CHANGELOG.md does not need updating

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -193,6 +193,21 @@ mergeStream(project.sources(), project.dependencies())
   .pipe(gulp.dest('build/'));
 ```
 
+### Custom Elements ES5 Adapter
+
+If you are serving ES5 custom elements (either from source or by compilation), it is critical to include the [Custom Elements ES5 Adapter](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js). This adapter provides compatibility between custom elements defined as ES5-style classes and browsers with native implementations of the Custom Elements API, such as Chrome. See the [adapter documentation](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js) for details of why this is necessary.
+
+The `project.addCustomElementsEs5Adapter()` method will return a transform stream that identifies your entry point HTML file and injects a block of HTML that loads the adapter when it is required.
```
Comment (FredKSchott) [inline README.md]: Change to match other sections format:

```
#### project.addCustomElementsEs5Adapter()

This method will return...
```

Comment (FredKSchott) [inline README.md]: Other comments:
- `entry point` -> `entrypoint`
- Does this stream now only change your entrypoint? Or does it still affect all files where import is found?

File: README.md [hunk 2]
```diff
@@ -193,6 +193,21 @@ mergeStream(project.sources(), project.dependencies())
   .pipe(gulp.dest('build/'));
 ```
 
+### Custom Elements ES5 Adapter
+
+If you are serving ES5 custom elements (either from source or by compilation), it is critical to include the [Custom Elements ES5 Adapter](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js). This adapter provides compatibility between custom elements defined as ES5-style classes and browsers with native implementations of the Custom Elements API, such as Chrome. See the [adapter documentation](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js) for details of why this is necessary.
```
Comment (FredKSchott) [inline README.md]: User question: Why wouldn't I just add the adapter myself? Why do I need a transform to do this for me? Am I missing something?

Comment (FredKSchott) [inline README.md]: Is there a concept of an ES5 custom element outside of compilation? Related, might be worth calling out Babel specifically here:

```
If you are using a JavaScript compilation tool like Babel to compile custom element classes, it is critical to include the...
```

File: README.md (hunk 1 above)
Comment (aomarks) [inline README.md]: Done.

Clarified.

File: README.md (hunk 2 above)
Comment (aomarks) [inline README.md]: Explained how you can use the stream to inject conditionally.

Mentioned Babel.

**Comment by aomarks** [comment]
PTAL

File: README.md [hunk 3]
```diff
@@ -193,6 +193,23 @@ mergeStream(project.sources(), project.dependencies())
   .pipe(gulp.dest('build/'));
 ```
 
+### Custom Elements ES5 Adapter
+
+If you are serving ES5 custom elements (either from source or by compilation with a tool like Babel), it is critical to include the [Custom Elements ES5 Adapter](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js). This adapter provides compatibility between custom elements defined as ES5-style classes and browsers with native implementations of the Custom Elements API, such as Chrome. See the [adapter documentation](https://github.com/webcomponents/custom-elements/blob/master/src/native-shim.js) for details of why this is necessary.
```
Comment (FredKSchott) [inline README.md]: The word "serving" just threw me off while re-reading, does "If you are building..." work here? Or "If your build outputs..."?

Comment (aomarks) [inline README.md]: How's this?

Comment (FredKSchott) [inline README.md]: 👍  looks great, make the change `serve` -> `output` here as well?

Comment (aomarks) [inline README.md]: Done.

**Pull request merged**